source/deploying-with-puppet/setup-puppet/install-puppet-master.rst

```diff
@@ -15,11 +15,16 @@ Install the Puppet yum repository and then the "puppet-server" package. See http
 Installation on Debian/Ubuntu
 ------------------------------
 
-Install the Puppet apt repository, and then the "puppetserver" package.  See https://apt.puppetlabs.com to find the correct deb file to install the puppet repo for your Linux distribution.  For example, with the Trusty release of Ubuntu, you would do the following::
+Install ``curl``, ``apt-transport-https`` and ``lsb-release``::
 
-   $ wget https://apt.puppetlabs.com/puppetlabs-release-pc1-trusty.deb
-   $ sudo dpkg -i puppetlabs-release-pc1-trusty.deb
-   $ sudo apt-get update && apt-get install puppetserver
+	$ sudo apt-get update
+	$ sudo apt-get install curl apt-transport-https lsb-release
+
+Get the adecuate Puppet apt repository, and then the "puppetserver" package. See https://apt.puppetlabs.com to find the correct deb file to install the puppet repo for your Linux distribution, you can use next script to make installation more silently::
+
+  $ wget "https://apt.puppetlabs.com/puppetlabs-release-pc1-$(lsb_release -cs).deb"
+  $ sudo dpkg -i "puppetlabs-release-pc1-$(lsb_release -cs).deb"
+  $ sudo apt-get update && apt-get install puppetserver
```

Comment:
Hi @mjflve, this is a good change, but add sudo to installation puppetserver command
change 
`sudo apt-get update && apt-get install puppetserver`

to 
`sudo apt-get update && sudo apt-get install puppetserver`